### DISCUSSION TITLE
Quasar Admin CRM New Version [Premium] - Built on Quasar 2/Vue 3

### DISCUSSION THREAD

**Discussion by mayank091193** [quasarframework/quasar#14665] [Show and Tell]
I have re-designed my Admin CRM template and given it a totally new look. You can buy the new version for $79. If you are interested, please email me at [mayank091193@gmail.com](mailto:mayank091193@gmail.com)

Live preview: https://quasar-admin-crm-premium.netlify.app/#/

Few Features:

- Modern and responsive design
- CRM Dashboard
- Analytics Dashboard
- Charts
- Calendar
- Export Feature

Please provide your valuable feedback and I will try to work on it. I will be adding a few more screens to the premium version in the near future.

Please find some screenshots of the new version below.

![Analytics](https://user-images.githubusercontent.com/27857088/196949590-11faec35-fc93-446f-99bb-f7af9401a2fe.png)

![CRM](https://user-images.githubusercontent.com/27857088/196949616-bfa2c87a-7a58-4f54-ad74-e5c3e2b07d28.png)

![Customer List](https://user-images.githubusercontent.com/27857088/196949639-458a9e6b-af12-453a-883b-f3687afff82b.png)

![Login](https://user-images.githubusercontent.com/27857088/196949663-b6c0491a-947d-4c34-ab94-2a9ddfb6bb42.png)

**Comment by ezeqqi** [comment]
Wow, such an amazing work. Congrats!
